### DISCUSSION TITLE
simple alias detection

### DISCUSSION THREAD

**Discussion by goerlibe** [discopop-project/discopop#341] [Development]
in `discopop/rtlib/CMakeLists.txt`:

```
# compile simple-alias-detection
exec_program(${CMAKE_CURRENT_SOURCE_DIR}/simple-alias-detection/compile.sh
    ARGS ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}
)
``` 

causes cmake configure message

```
running /home/bertingoerlich/disco-stack/discopop/rtlib/simple-alias-detection/compile.sh /home/bertingoerlich/disco-stack/discopop/rtlib /home/bertingoerlich/disco-stack/build/discopop/rtlib  2>&1
/bin/sh: 1: /home/bertingoerlich/disco-stack/discopop/rtlib/simple-alias-detection/compile.sh: not found
``` 

because the `compile.sh` file does not exist.

The build works fine but this does not seem right... Are you maybe using the `compile.sh` file locally and it was never checked into version control? @lukasrothenberger

**Comment by lukasrothenberger** [comment]
Hi @goerlibe

> Are you maybe using the compile.sh file locally and it was never checked into version control?

Jup, should be fixed in #342

Thank you very much!
